runtime/compiler/runtime/MethodMetaData.c

```diff
@@ -2127,15 +2134,21 @@ UDATA getCurrentByteCodeIndexAndIsSameReceiver(J9TR_MethodMetaData * methodMetaD
          prevInlinedCallSite = inlinedCallSite;
          inlinedCallSite = getNextInlinedCallSite(methodMetaData, inlinedCallSite);
          }
-      byteCodeInfo = (TR_ByteCodeInfo *)getByteCodeInfo(prevInlinedCallSite);
-      if (inlinedCallSite)
+      if (prevInlinedCallSite)
+         byteCodeInfo = (TR_ByteCodeInfo *)getByteCodeInfo(prevInlinedCallSite);
+      else if (inlinedCallSite)
```

Comment:
I don't think this logic is functionally equivalent to the way it was originally.  I think the `else` needs to be removed to maintain that equivalence.

Reply:
That's correct. I will remove the `else`.